cli/command/stack/deploy.go

```diff
@@ -20,7 +20,7 @@ func newDeployCommand(dockerCli command.Cli) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			opts.Namespace = args[0]
 			if dockerCli.ClientInfo().HasKubernetes() {
-				kli, err := kubernetes.WrapCli(dockerCli, cmd)
+				kli, err := kubernetes.WrapCli(dockerCli, kubernetes.NewOptions(cmd.Flags()))
```

Comment:
I hope that in the future when we add kube to more commands we can have this accept an `Options` struct instead of `cmd.Flags()`.

Reply:
Ok I'll refactor that in a future PR.